LayoutTests/imported/w3c/web-platform-tests/html/canvas/offscreen/manual/filter/offscreencanvas.filter-expected.txt

```diff
@@ -1,3 +1,3 @@
 
-PASS none
+FAIL none assert_array_equals: The image data generated by filter blur(10px) should be the same for both OffscreenCanvas and regular canvas expected property 0 to be 113 but got 0 (expected array object "113,38,28,27,111,34,26,30,108,39,23,33,106,43,28,36,105,46,26,39" got object "0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0")
```

Comment:
This is a progression. The filter in both the detached and the offscreen canvases was not working. So `getImageData()` from both canvases was retuning zeros and this is why the test was passing.

With this PR, the detached canvas is working. When the Offscreen+filter is fixed in https://github.com/WebKit/WebKit/pull/30074, the test will pass again.